### PR TITLE
Adding another playground page to show a less CoffeeMaker-like example

### PR DESCRIPTION
**Pull request by mikelikespie**

### CODE REVIEW

File: CleansePlayground.playground/Pages/GithubClientExample.xcplaygroundpage/Contents.swift [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+ will list a User's projects
+*/
+
+import Cleanse
+
+import XCPlayground
+
+
+//: Since we will be doing an asynchronous call, we need to make it so the execution doesn't stop immediately
+XCPlaygroundPage.currentPage.needsIndefiniteExecution = true
```
Comment (dfed) [inline CleansePlayground.playground/Pages/GithubClientExample.xcplaygroundpage/Contents.swift]: TIL!

File: CleansePlayground.playground/Pages/GithubClientExample.xcplaygroundpage/Contents.swift [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+//: This is the base URL for github
```
Comment (dfed) [inline CleansePlayground.playground/Pages/GithubClientExample.xcplaygroundpage/Contents.swift]: curious what `//:` does? Is it the same as `///`?

File: CleansePlayground.playground/Pages/GithubClientExample.xcplaygroundpage/Contents.swift [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+                    delegate: nil,
+                    delegateQueue: NSOperationQueue.mainQueue()
+                )
+            }
+    }
+}
+
+//: Simple Service that lists "Member" logins for the current organization
+protocol GithubListMembersService {
+    func listMembers(organizationName: String, handler: [String] -> ())
```
Comment (dfed) [inline CleansePlayground.playground/Pages/GithubClientExample.xcplaygroundpage/Contents.swift]: style nit (feel free to ignore): I like to name the parameters to my handlers for easier legibility

File: CleansePlayground.playground/Pages/GithubClientExample.xcplaygroundpage/Contents.swift [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+
+    /// Lists members of an organization
+    func listMembers(organizationName: String, handler: [String] -> ()) {
+
+        let url = githubURL
+            .get()
+            .URLByAppendingPathComponent("orgs/\(organizationName)/public_members")
+
+        let dataTask = urlSession.dataTaskWithURL(url) { data, response, error in
+//: For simplicity of example, just yield an empty list if the request wasn't successful
```
Comment (dfed) [inline CleansePlayground.playground/Pages/GithubClientExample.xcplaygroundpage/Contents.swift]: lacking leading whitespace

File: CleansePlayground.playground/Pages/GithubClientExample.xcplaygroundpage/Contents.swift (hunk 2 above)
Comment (mikelikespie) [inline CleansePlayground.playground/Pages/GithubClientExample.xcplaygroundpage/Contents.swift]: For a playground it makes nice formatting

https://developer.apple.com/library/ios/documentation/Xcode/Reference/xcode_markup_formatting_ref/Headings.html#//apple_ref/doc/uid/TP40016497-CH8-SW1

githubclientexample_xcplaygroundpage

File: CleansePlayground.playground/Pages/GithubClientExample.xcplaygroundpage/Contents.swift (hunk 4 above)
Comment (mikelikespie) [inline CleansePlayground.playground/Pages/GithubClientExample.xcplaygroundpage/Contents.swift]: Leading whitespace makes it not render in playground

File: CleansePlayground.playground/Pages/GithubClientExample.xcplaygroundpage/Contents.swift (hunk 2 above)
Comment (dfed) [inline CleansePlayground.playground/Pages/GithubClientExample.xcplaygroundpage/Contents.swift]: hawt!

File: CleansePlayground.playground/Pages/GithubClientExample.xcplaygroundpage/Contents.swift [hunk 5]
```diff
@@ -0,0 +1,126 @@
+/*:
+ 
+ ## Github Client Example.
+
+ This is a demonstration of using Cleanse and NSURLSession and friends to configure a github client that
```
Comment (swankjesse) [inline CleansePlayground.playground/Pages/GithubClientExample.xcplaygroundpage/Contents.swift]: Nice example.

File: CleansePlayground.playground/Pages/GithubClientExample.xcplaygroundpage/Contents.swift (hunk 1 above)
Comment (mikelikespie) [inline CleansePlayground.playground/Pages/GithubClientExample.xcplaygroundpage/Contents.swift]: SO I think that's enabled by default if you use a liveView FWIW

File: CleansePlayground.playground/Pages/GithubClientExample.xcplaygroundpage/Contents.swift (hunk 5 above)
Comment (mikelikespie) [inline CleansePlayground.playground/Pages/GithubClientExample.xcplaygroundpage/Contents.swift]: Thanks!

File: CleansePlayground.playground/Pages/GithubClientExample.xcplaygroundpage/Contents.swift (hunk 3 above)
Comment (mikelikespie) [inline CleansePlayground.playground/Pages/GithubClientExample.xcplaygroundpage/Contents.swift]: Will address this in followup. Have like 10 min to make blog post

**Comment by swankjesse** [comment]
+1

**Comment by dfed** [comment]
LGTM.

**Pull request merged**